proposals/array_append.json

```diff
@@ -23,6 +23,18 @@
             "schema": {
                 "description": "Any data type is allowed."
             }
+        },
+        {
+            "name": "label",
+            "description": "If the given array is a labeled array, a new label for the new value should be given. If not given or `null`, the array index as string is used as the label. If in any case the label exists, a `LabelExists` exception is thrown.",
```

Comment:
> a new label for the new value should be given.

the parameter is optional, so I would replace "should" with "can"

Reply:
Should != must. Here should is meant to be understood as a recommendation so that you don't end up with weird string indices.